src/DIRAC/Core/Utilities/Graphs/QualityMapGraph.py

```diff
@@ -168,10 +168,10 @@ def draw(self):
         setp(self.ax.get_yticklines(), markersize=0.0)  # pylint: disable=not-callable
 
         cax, kw = make_axes(self.ax, orientation="vertical", fraction=0.07)
-        cb = ColorbarBase(
+        ColorbarBase(
```

Comment:
Does this statement still do anything, or just create an object that is not used?

Reply:
This statement is actually still useful, I just made a quick test in the certification environment: 

- Without it:
![image](https://github.com/DIRACGrid/DIRAC/assets/19813740/8ca08d12-f781-4d47-b6ec-0844dfe0dedb)

- With it:
![image](https://github.com/DIRACGrid/DIRAC/assets/19813740/27582d0e-d273-4272-b6e6-bde896ba16be)


But the line that I replaced seems to have no effect, I have not spotted any change when I removed it: this is true for both `cb.draw_all()` and `self.figure.draw_without_rendering()`.
I would let it here for now as it might affect something I can't see.